key-tester.js

```diff
@@ -126,6 +126,7 @@ function run(form_url, ga_code) {
             $.getJSON(url, function(data) {
                 var key = data[0].key;
                 $("#form-ssh-key").val(key);
+                check_ssh_key();
```

Comment:
`check_ssh_key` is already called in the `.change` handler so you can also trigger it (`$('#form-ssh-key').trigger('change')`), not sure if it's better though.